content/golanguk-conference.md

```diff
@@ -0,0 +1,32 @@
++++
+author = ["Alexandre González"]
+date = "2015-03-27T20:30:00-00:00"
+linktitle = "The Golang UK Conference"
```

Comment:
"Go"

golang is just a helpful tag for searches. like rustlang, etc. It's not the name of the language.